StompClientLib/Classes/StompClientLib.swift

```diff
@@ -97,34 +110,41 @@ public class StompClientLib: NSObject, SRWebSocketDelegate {
     
     public func openSocketWithURLRequest(request: NSURLRequest, delegate: StompClientLibDelegate, connectionHeaders: [String: String]?) {
         self.connectionHeaders = connectionHeaders
-        openSocketWithURLRequest(request: request, delegate: delegate)
-        self.connection = true
```

Comment:
Why did you remove the change the status of connection? Don't you need to set the status of change?